cellrank/tl/_transition_matrix.py

```diff
@@ -23,9 +20,9 @@ def transition_matrix(
     xkey: str = "Ms",
     conn_key: str = "connectivities",
     gene_subset: Optional[Iterable] = None,
-    mode: str = VelocityMode.DETERMINISTIC.s,
-    backward_mode: str = BackwardMode.TRANSPOSE.s,
-    scheme: str = Scheme.CORRELATION.s,
+    mode: str = VelocityMode.DETERMINISTIC,
+    backward_mode: str = BackwardMode.TRANSPOSE,
+    scheme: str = Scheme.CORRELATION,
```

Comment:
```suggestion
    mode: VelocityMode = VelocityMode.DETERMINISTIC,
    backward_mode: BackwardMode = BackwardMode.TRANSPOSE,
    scheme: Scheme = Scheme.CORRELATION,
```

or using `Literal[...]`.

Reply:
Done (2nd version).